Expensify/App
# Refactor `TextInputWithFocusStyles` with TextInput

### Details

- [x] **TextInputWithFocusStyles**
   #### Purpose of component
   1. This is just used to change some styles when input is in a focused state and vice-versa. 
   
  Remove the extra component and use TextInput instead.
   
### Fixed Issues

$ https://github.com/Expensify/App/issues/6584
$ https://github.com/Expensify/App/issues/7309

### Tests | QA Steps
1. Open Request Money page. 
2. Click Currency.
3. Check that you are able to search the currency from the search bar. 


### Tested On


### Screenshots


#### Web | Desktop

![image](https://user-images.githubusercontent.com/24370807/153768952-70477cde-6f94-4761-9446-05c3e5b43366.png)

#### Mobile Web



#### iOS

![image](https://user-images.githubusercontent.com/24370807/153769228-5245a5ed-b961-4a17-8934-fa64bda6b818.png)

#### Android

![image](https://user-images.githubusercontent.com/24370807/153900861-d7e3f4ff-9cf9-4885-bb9e-bbc38704262c.png)
